docs/core-study-details.md

```diff
@@ -8,98 +8,501 @@ nav_order: 5
- 
-Patient count can be the 
-- number of **unique patients**
-- number of **unique patient encounters**
-- number of **unique patient encounters with documented medical note**
-- Other types of counts are also possible, these are the most common.
+The core study aims to provide the following features:
 
-Example
+- **FHIR US Core V4 Profile data** - For resources that are supported by Cumulus, we attempt to extract all required/extensible fields from a FHIR dataset, if present
+- **Flattened FHIR resources** - The core study provided flattened tables from nested FHIR resources, making queries easier to construct without having to worry about the nuances of conditional unnesting against potentially missing data
+- **Summary Exports** - The core study will provide some basic count data across the entire cohort that you've processed via ETL, which can be useful as a verification step for validating the integrity of the data you're extracting from your EHR system
 
-    #count (total) =
-    #count patients age 9 =
-    #count patients age 9 and rtPCR POS =
-    #count patients age 9 and rtPCR NEG =
-    #count rtPCR POS =
-    #count rtPCR NEG =
+If you are authoring a study, and are focused only on clinical analysis (i.e. you aren't working on data quality/data governance issues), we :strongly: recommend you use the core study as the starting point for your own work. See 
+[Creating Studies](./creating-studies.md)
+for more information.
 
-[SQL CUBE](https://prestodb.io/docs/current/sql/select.html#group-by-clause) produces a "[Mathematical Power Set](http://en.wikipedia.org/wiki/Power_set)" for every patient subgroup.  
-These numbers are useful inputs for maths that leverage [Joint Probability Distributions](https://en.wikipedia.org/wiki/Joint_probability_distribution). 
+# Table format
 
-Examples: 
+## core count tables
 
-- [Odds Ratio](https://en.wikipedia.org/wiki/Odds_ratio) of patient group A vs B 
-- [Relative Risk Ratio](https://en.wikipedia.org/wiki/Relative_risk) of patient group A vs B
-- [Chi-Squared Test](https://en.wikipedia.org/wiki/Chi-squared_test) significance of difference between patient groups
-- [Entropy and Mutual Information](https://en.wikipedia.org/wiki/Mutual_information) (core information theory measures) 
-- [Decision Tree](https://en.wikipedia.org/wiki/Decision_tree) sorts patients into different predicted classes, with visualized tree   
-- [Naive Bayes Classifier](https://en.wikipedia.org/wiki/Naive_Bayes_classifier) very fast probabilistic classifier
-- others
+### core__count_condition_month
+|      Column      | Type  |Description|
+|------------------|-------|-----------|
+|cnt               |bigint |count      |
+|cond_category_code|varchar|Encounter Code (Healthcare Setting)|
+|cond_month        |varchar|Month condition recorded|
+|cond_code_display |varchar|Condition code|
 
-## Core study exportable counts tables
+### core__count_documentreference_month
+|     Column      | Type  |Description|
```

Comment:
When I was just playing with the previous version of this file, I noticed that the resulting HTML from jekyll that gets put on the doc site required a newline between any section header and the table. Worth testing, but I'd have your generator add a line to be safe.

Reply:
oh dear - ok, this might take some work.

Reply:
Ok, it's a bit hacky, but that's the default behavior.